frontend/src/component/common/NewConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/ResolveInput/ResolveInput.tsx

```diff
@@ -84,23 +82,11 @@ export const ResolveInput = ({
     removeValue,
     error,
 }: IResolveInputProps) => {
-    const useNewLegalValueInput = useUiFlag('addEditStrategy');
     const resolveInput = () => {
         switch (input) {
             case IN_OPERATORS_LEGAL_VALUES:
             case STRING_OPERATORS_LEGAL_VALUES:
-                return useNewLegalValueInput ? (
-                    <LegalValuesSelector
-                        data={resolveLegalValues(
-                            constraintValues,
-                            contextDefinition.legalValues,
-                        )}
-                        constraintValues={constraintValues}
-                        values={localConstraint.values || []}
-                        setValuesWithRecord={setValuesWithRecord}
-                        setValues={setValues}
-                    />
-                ) : (
```

Comment:
This is not needed. The new component should only be used by the editable constraint for now, which uses it directly. Moreover, the project actions filter component doesn't have legal values, so it's not relevant there.